lib/tokenize.cpp

```diff
@@ -9250,8 +9250,10 @@ void Tokenizer::simplifyKeyword()
                 Token::createMutualLinks(braceStart, braceEnd);
             }
 
-            // 3) thread_local
+            // 3) thread_local -> static
+            //    on single thread thread_local has the effect of static
             else if (tok->str() == "thread_local") {
+                tok->insertToken("static", "thread_local");
```

Comment:
I would suggest:

    tok->str("static");
    tok->originalName("thread_local");

Reply:
I changed it to
```C++
tok->originalName(tok->str());
tok->str("static");
```